### DISCUSSION TITLE
question re install from another firmware

### DISCUSSION THREAD

**Discussion by tico74** [mriscoc/Ender3V2S1#753] [Q&A]
Hi All,

Have been using another popular custom firmware and it has been good but I am now wanting to try Mriscoc on my ender 3v2 with BL touch.

The question I have is once I have updated everything to Mriscoc (including display screen) do I need to adjust anything in Cura to reflect the new firmware .i.e. I remember following a YouTube video when I used the other firmware and I had to change a few things in the G-code.

Is there anything extra I am required to do to G-code or.....
 will I need to re do the extruder steps?

![example](https://user-images.githubusercontent.com/102801748/220868152-3a56e25b-239b-4ed7-a8a1-f3c73364f9da.jpg)

**Comment by Jimidean23** [comment]
Yes. Read entire article.  Some useful info in the bottom.  

https://github.com/mriscoc/Ender3V2S1/wiki/Slicer-G-code-Scripts

**Reply by tico74** [reply]
thanks...I'll have a read and see if I can makes sense of it.

**Reply by tico74** [reply]
I have had a look and the Cura "start script" differs quite a bit. I have the opportunity to copy and paste it onto Cura but I am concerned that I will completely stuff up all my settings. 
I am aware I can copy and save it and revert back to it but is this the correct thing to do?

Fisrt pic is from mriscoc link . second pic is my current setup from cura.

Can some possibly advise.

regards


![slicer g code from mriscoc link](https://user-images.githubusercontent.com/102801748/221126349-3b1b235a-6815-473a-aa2e-31def1a0b946.jpg)
![cura setup when using Jyers on cura](https://user-images.githubusercontent.com/102801748/221126502-4029f297-7b56-43b3-8b15-f387e5b38b93.jpg)
